src/Api/Library/Shared/Communicate/Email.php

```diff
@@ -14,7 +16,7 @@ class Email
     public static function send($from, $to, $subject, $content, $htmlContent = '')
     {
         // Create the Transport
-        $transport = \Swift_SmtpTransport::newInstance('mail', 25);
```

Comment:
I note that you removed the port number from the constructor, however I checked the [source code](https://github.com/swiftmailer/swiftmailer/blob/master/lib/classes/Swift/SmtpTransport.php#L33) and this parameter has a default of 25 as expected, so good to go.

Reply:
yes, that's why I did it, thanks for double-checking.